client/components/choose-player-menu.jsx

```diff
@@ -39,8 +37,8 @@ ChoosePlayerMenu = React.createClass({
 
   render() {
 
-    var menuItems = this.data.players.map((player) => {
-      return { title: player.name };
+    let menuItems = this.data.players.map((player) => {
```

Comment:
@dshvazz Are you just experimenting with `let` here or is there a scoping reason to do it?